Documentation/ceph-upgrade.md

```diff
@@ -60,6 +60,12 @@ any time without compatibility support and without prior notice.
 was the last Rook release which will support Ceph's Luminous (v12.x.x) version. These are the only
 supported major versions of Ceph.
 
+Rook documentation for 1.1 has identified some Ceph configuration options that the user is highly
+advised to consider. See the docs [here](ceph-configuration.md#default-pg-and-pgp-counts). It is
+highly advised to consider these now before upgrading Rook as well. If the user determines that
```

Comment:
- nit: "highly advised" is mentioned twice. Do we need to be that alarming?
- I wonder if this is better to recommend as a post-upgrade consideration. At that point they will be able to use the new cluster CR option for enabling the pg autoscaler.

Reply:
It is a post-upgrade consideration, but it's mentioned here to prepare the user. I should probably update this to mention the mgr module enabling options.